samples/fuel_gauge/max17048/README.rst

```diff
@@ -40,12 +42,10 @@ Sample output
 ```
 *** Booting Zephyr OS build 16043f62a40a ***
 Found device "max17048@36", getting fuel gauge data
-Time to empty 1911
-Time to full 0
+Polling for fuel gauge data...
+----------------------------------------
 Charge 72%
-Voltage 3968
+Voltage 3968 uV
 Time to empty 1911
 Time to full 0
-Charge 72%
-Voltage 3968
```

Comment:
Either the API is wrong, or this value makes no sense. The measurement should be in microvolts (according to the Doxygen comment of the property), but both mine and the old one are in mV.